pubnub/pubnub.py

```diff
@@ -29,8 +29,8 @@ class PubNub(PubNubCore):
     def __init__(self, config):
         assert isinstance(config, PNConfiguration)
 
-        self._request_handler = RequestsRequestHandler(self)
         PubNubCore.__init__(self, config)
+        self._request_handler = RequestsRequestHandler(self)
```

Comment:
Does moving this do anything useful?

Reply:
Yes, PubNubCore assigns `pubnub.config = config` so that self can access the `config.origin` in `self._request_handler = RequestsRequestHandler(self)`

Reply:
It allows access to the `origin` configured by customer `self.session.mount('http://%s' % pubnub.config.origin, HTTPAdapter(max_retries=1, pool_maxsize=500))`